File: Tugas2/python/ram.py
class ram():
    __capacityr = ""
    __pricer = 0

    # construct
    def __init__(self, capacityr = "", pricer = 0):
        self.__type = type
        self.__capacityr = capacityr
        self.__pricer = pricer

    # set dan get
    def setcapacityr(self, capacityr):
        self.__capacityr = capacityr
    
    def getcapacityr(self):
        return self.__capacityr

    def setpricer(self, pricer):
        self.__pricer = pricer
    
    def getpricer(self):
        return self.__pricer

    # output
    def printram(self):
        print("capacity : " + str(self.__capacityr))
        print("price : " + str(self.__pricer))

File: Tugas1/ram.cpp
#include <iostream>
#include <string>

using namespace std;

class ram{
	private:
		string capacityr;
        int pricer;
	
	public:
		//Construct
		ram(){}
		
		ram(string capacityr, int pricer){
			this->capacityr = capacityr;
			this->pricer = pricer;
		}
		
		//Set dan Get
		void setcapacityr(string capacityr)
        {
            this->capacityr = capacityr;
        }
		string getcapacityr(){
			return this->capacityr;
		}
		
		void setpricer(int pricer){
			this->pricer = pricer;
		}
		int getpricer(){
			return this->pricer;
		}

        void printram(){
            cout << "Kapasitas RAM : " << getcapacityr() << endl;
            cout << "Harga RAM     : " << getpricer() << endl;
        }		
		
		~ram(){
		}
};